core/base/document.php

```diff
@@ -734,6 +734,9 @@ protected function get_initial_config() {
 			if ( Plugin::$instance->experiments->is_feature_active( 'atomic_widgets' ) ) {
 				$container_config['div-block'] =
 					Plugin::$instance->elements_manager->get_element_types( 'div-block' )->get_config();
+
+				$container_config['flexbox'] =
+					Plugin::$instance->elements_manager->get_element_types( 'flexbox' )->get_config();
```

Comment:
Maybe?
```suggestion
    $elements = [ 'div-block', 'flexbox' ];
    
    foreach ( $elements as $element ) {
        $container_config[$element] = Plugin::$instance->elements_manager->get_element_types( $element )->get_config();
    }
```
Not crucial, just feels like some of the code is duplicated.